website/core/views.py

```diff
@@ -76,4 +71,4 @@ def state_biz(state_code=None):
 
 @core.route('/State_list')
 def State_list():
```

Comment:
We might need to force push onto master because this is the most up to date code with Nicks changes. I still havent done anything with the google sheets api to bring in any data from the excel sheet myself.

Reply:
go ahead and force push it then. no big deal

Reply:
lets go over the changes together before we do it tho I want us to be on the same page all the way. and we can do it tomorrow if you prefer. Also any work that you have just make a pr for it and then we can still merge it to master. or if there is problems with that too, we might have to merge it with this branch.

Reply:
sounds good. I'll call you tomorrow then.